src/test/java/salesforce/opoortunity/OpportunityTests.java

```diff
@@ -0,0 +1,51 @@
+package salesforce.opoortunity;
+
+import org.testng.annotations.Test;
+import salesforce.base.BaseTest;
+import salesforce.config.ConfigEnvVar;
+import salesforce.ui.PageTransporter;
+import salesforce.ui.pages.HomePage;
+import salesforce.ui.pages.LoginPage;
+import salesforce.ui.pages.opportunity.NewOpportunityPage;
+import salesforce.ui.pages.opportunity.OpportunityPage;
+
+public class OpportunityTests extends BaseTest {
+
+    private OpportunityPage opportunityPage;
+    private NewOpportunityPage formOpportunity;
+    private HomePage homePage;
+    private PageTransporter pageTransporter = new PageTransporter();
+
+    @Test
+    public void testCreateOpportunityWithRequiredValuesOnly() {
+        String username = ConfigEnvVar.getInstance().getUserName();
```

Comment:
Maybe you can put these configurations in the before method

Reply:
its already fix